sdk/core/core-paging/samples-dev/getPagedAsyncIteratorSample.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample builds a simple paging operation using getPagedAsyncIterator
+ *
+ * @summary builds a simple paging operation using getPagedAsyncIterator
+ * @azsdk-weight 40
+ */
+
+import { getPagedAsyncIterator, PagedResult, PageSettings } from "@azure/core-paging";
+
+type IsAny<T> = boolean extends (T extends never ? true : false) ? true : false
```

Comment:
This is quite clever.

Reply:
Love seeing a novel any test 😁 I usually roll with `0 extends (1 & T) ? true : false`.